tools/linter/excluded-apis.txt

```diff
@@ -0,0 +1,93 @@
+abusiveexperiencereport
```

Comment:
Looking through the generated CSV, looks like we should also exclude,

- classroom
- cloudshell
- cloudsearch
- cloudbilling
- script
- vault
- vision